apps/gruenbuch/models.py

```diff
@@ -0,0 +1,128 @@
+from django.db import models
+from wagtail.admin.edit_handlers import FieldPanel
+from wagtail.admin.edit_handlers import ObjectList
+from wagtail.admin.edit_handlers import StreamFieldPanel
+from wagtail.admin.edit_handlers import TabbedInterface
+from wagtail.core import blocks
+from wagtail.core import fields
+from wagtail.core.models import Page
+
+from apps import blocks as apps_blocks
+from apps.contrib.translations import TranslatedField
+
+
+class GruenbuchIndexPage(Page):
+    template = 'gruenbuch_page.html'
+
+    @property
+    def book(self):
+        return self
+
+    @property
+    def chapters(self):
+        return self.get_children().specific()
+
+    @property
+    def active_chapter(self):
+        return self.chapters.first()
+
+    @property
+    def next_chapter(self):
+        if self.get_next_sibling():
```

Comment:
Why does the index page have a next sibling? And if it does, that could be any page, right? So, we don't want to return that? 
(and did you check using `active-chapter.get_next_sibling` in the template? Then you don't need this property here and also not on the gruenbuch-detail.)

Reply:
Oh, or does using `get_next_sibling` in the template not work because you need the `specific` for the property in the detail?